packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -580,7 +615,8 @@ export class Linker extends Component<LinkerProps, LinkerState> {
 
   isOverlayShown(): boolean {
     const { activeTool } = this.props;
-    return activeTool === ToolType.LINKER;
+    const { isDraggingPanel } = this.state;
```

Comment:
I didn't realize we don't want the fade transition for this case, so it should be a separate boolean from `isOverlayShown()`.
Just wrap the whole CSSTransition in `!isDraggingPanel ? <CSSTransition in={isOverlayShown} ... > : null`, then you'll have a fade when activating/deactivating the linker by changing the tool, but it'll still be instant when dragging the panel.

Even with these changes, it looks like Linker gets a couple renders in before receiving the `.DRAGGING` event and still logs the error a couple time.  I think logging an error when trying to draw the link may be a bit aggressive. Changing that to `log.warn` instead may be useful: https://github.com/deephaven/web-client-ui/blob/f5794d797403eb3ab2c553d237de11171ac7c9e6/packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.tsx#L291